frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/PreviewPane.styled.tsx

```diff
@@ -1,9 +1,20 @@
 import styled from "@emotion/styled";
+import { css } from "@emotion/react";
 
-export const PreviewPaneContainer = styled.div<{ isTransparent?: boolean }>`
+export const PreviewPaneContainer = styled.div<{
+  isTransparent?: boolean;
+  hidden?: boolean;
+}>`
   width: 100%;
   min-height: 17.5rem;
 
+  ${({ hidden }) =>
+    hidden &&
+    css`
+      visibility: hidden;
```

Comment:
Can't use `display: none` because it will cause the iframe to be reloaded again. So I need the iframe to stay in the dom. I set `position: absolute` so it doesn't take up space inside the modal when switching to other tabs.